prisma/schema.prisma

```diff
@@ -33,6 +33,7 @@ model DnsRecord {
   expiresAt    DateTime
   lastNotified DateTime?
   user         User            @relation(fields: [username], references: [username], onDelete: Cascade)
+  challenge    Challenge[]
```

Comment:
This feels off to me.  Can you explain this?  Is there no where else we can store this relation?  It feels odd for a record to reference other records.  Even putting on the User would feel a bit better.

Reply:
What do you think?

Reply:
That makes sense to me.

Reply:
Okay, I'll refactor